CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+# 0.8.2 (2019-04-11)
+
+## Migration Disclaimer
+
+If you have the following set in your M3Query or M3Coordinator config, you should remove it:
```

Comment:
Can you explain why? This sounds pretty vague on its own. How would users have ended up with this config? What relevance does it have? Why do we need to change it?

The migration disclaimer for 0.7.0 commit log format changes is a helpful example I think